packages/amesos2/src/Amesos2_EpetraRowMatrix_AbstractMatrixAdapter_decl.hpp

```diff
@@ -151,6 +151,11 @@ namespace Amesos2 {
     const RCP<const Tpetra::Map<local_ordinal_t,
 				global_ordinal_t,
 				node_t> >
+    getMap_impl() const;
```

Comment:
Same as before .. Comments from previous review is still there .. May be this is needed ?